chcemvediet/apps/inforequests/tests/test_models/test_branch.py

```diff
@@ -366,84 +361,84 @@ def test_can_add_x_properties(self):
         tests = (
                 (Action.TYPES.REQUEST, Bunch(
                     scenario=[],
-                    allowed=[           u'confirmation', u'extension', u'advancement', u'clarification_request', u'disclosure', u'refusal', u'obligee_action', u'obligee_email_action'],
-                    expired=[u'appeal', u'confirmation', u'extension', u'advancement', u'clarification_request', u'disclosure', u'refusal', u'obligee_action', u'obligee_email_action', u'applicant_action'],
+                    allowed=[           u'confirmation', u'extension', u'advancement', u'clarification_request', u'disclosure', u'refusal', u'disclosure_refusal_advancement_or_extension', u'obligee_action', u'obligee_email_action'],
+                    expired=[u'appeal', u'confirmation', u'extension', u'advancement', u'clarification_request', u'disclosure', u'refusal', u'disclosure_refusal_advancement_or_extension', u'obligee_action', u'obligee_email_action', u'applicant_action'],
                     )),
                 (Action.TYPES.CLARIFICATION_RESPONSE, Bunch(
                     scenario=[u'clarification_request', u'clarification_response'],
-                    allowed=[           u'extension', u'advancement', u'clarification_request', u'disclosure', u'refusal', u'obligee_action', u'obligee_email_action'],
-                    expired=[u'appeal', u'extension', u'advancement', u'clarification_request', u'disclosure', u'refusal', u'obligee_action', u'obligee_email_action', u'applicant_action'],
+                    allowed=[           u'extension', u'advancement', u'clarification_request', u'disclosure', u'refusal', u'disclosure_refusal_advancement_or_extension', u'obligee_action', u'obligee_email_action'],
```

Comment:
@martinmacko47 Chceme testovat aj properties `can_add_disclosure_refusal_advancement_or_extension` a `can_add_remandment_affirmation_or_reversion`, kedze to su iba odvodeniny od zakladnych `can_add_*`?

Reply:
Kludne mozeme. Tak ako si to spravil je OK.